broadinstitute/consent
# DUOS-1260[risk=low]Create new get all users endpoint in UserResource with RoleName path param

SCOPE:
- New API for SO an Admin in User Resource class
- Deprecate old API in DACUser Resource class
- New service class method in User Service
- Deprecate old service class method in User Service
- New DAO call getUsersByInstitution
- tests for all new methods
- update API docs to include documentation of new API

ADDRESSES:
https://broadworkbench.atlassian.net/browse/DUOS-1260

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment
